### ISSUE TITLE
Add a scale setter to affect scaleX and scaleY easily.

### ISSUE THREAD

**Issue by lannymcnie** [CreateJS/EaselJS#698]
Currently you have to set both scaleX and scaleY separately. A `scale` setter would be a handy shortcut.

Note that it would be slower than the properties. We would want to document that.

**Comment by wdamien** [comment]
How about a new method instead?  ```displayObject.scale(x, y);``` it could be overloaded so if a single parameter is passed it affects both scaleX and scaleY.

**Comment by ylazy** [comment]
A new method would better: mc.scale(x, y); mc.resize(w, h);

**Comment by lannymcnie** [comment]
The idea of a scale setter, with a related getter is a cool idea (which would probably just return one of the scaleX/Y values, since they would need to be the same for it to work.

What would you expect `resize` to do? Would that set a scaleX/Y to a specific pixel size given the original bounds? I think that could be valuable.

**Comment by gskinner** [comment]
I would favour a property. If we add a `scale()` method, then I would also expect `translate()`, `rotate()`, `skew()`, which we already provide on `Matrix2D`, and which would not necessarily behave exactly the same between the two classes. It would also introduce additional documentation and maintenance requirements, and will lead to best practice confusion (the properties will remain best practice, since they will be faster).

Also worth noting that `scale()` and `.scale` would behave differently. The method would need to be multiplicative (ex. if `myThing.scaleX = 2`, and I call `myThing.scale(3)`, then `myThing.scaleX == 6`).

In short, `.scale` would be a shortcut, adding `scale()` would be a whole new transformation model.

Worth a mention as well: `setTransform(0,0,2,2)`.

**Comment by danzen** [comment]
I'm all for a scale property - I use a zim.scale(obj, scale) to get around it and also added a single scale property to zim.animate(), zim.move() and zim.Parallax() - been breathing a sigh of relief ever since.
